components/ToastMessage/assets/index.js

```diff
@@ -0,0 +1,9 @@
+const action = require('./action.png');
+const success = require('./success.png');
+const error = require('./error.png');
+
+export const images = {
+  action,
+  success,
+  error,
+};
```

Comment:
Is there a reason we're using PNGs instead of SVG icons?

Reply:
i can change. Not sure why i exported pngs from figma

Reply:
+1 for changing to svgs if possible